pkgs/development/ocaml-modules/uutf/default.nix

```diff
@@ -1,37 +1,26 @@
-{ stdenv, buildOcaml, fetchurl, ocaml, findlib, ocamlbuild, opam, cmdliner}:
+{ stdenv, fetchurl, ocaml, findlib, ocamlbuild, opam, cmdliner , topkg, uchar }:
 let
   pname = "uutf";
   webpage = "http://erratique.ch/software/${pname}";
 in
 
-buildOcaml rec {
-  name = pname;
-  version = "0.9.4";
-
-  minimumSupportedOcamlVersion = "4.00.0";
+stdenv.mkDerivation rec {
```

Comment:
I guess the idea of `  buildOcaml`   is to minimize code duplication by sharing common code, but it is very little in the case of ocaml, since seemingly every package has a different build process.